core/src/main/resources/defaults/blueprints/hdp30-edw-standalone.bp

```diff
@@ -0,0 +1,216 @@
+{
+  "description": "Useful for EDW analytics using Hive LLAP",
+  "blueprint": {
+    "Blueprints": {
+      "blueprint_name": "hdp30-edw-standalone-v1",
+      "stack_name": "HDP",
+      "stack_version": "3.0"
+    },
+    "settings": [
+      {
+        "recovery_settings": []
+      },
+      {
+        "service_settings": [
+          {
+            "name": "HIVE",
+            "credential_store_enabled": "false"
+          }
+        ]
+      },
+      {
+        "component_settings": []
+      }
+    ],
+    "configurations": [
+      {
+        "hive-interactive-env": {
+          "enable_hive_interactive": "true"
+        }
+      },
+      {
+        "core-site": {
+          "fs.trash.interval": "4331"
+        }
+      },
+      {
+        "hdfs-site": {
+          "dfs.namenode.safemode.threshold-pct": "0.99"
+        }
+      },
+      {
+        "hive-interactive-site": {
+          "hive.exec.orc.split.strategy": "BI",
+          "hive.stats.fetch.bitvector": "true",
+          "hive.metastore.rawstore.impl": "org.apache.hadoop.hive.metastore.cache.CachedStore"
+        }
+      },
+      {
+        "hive-site": {
+          "hive.exec.compress.output": "true",
+          "hive.merge.mapfiles": "true",
+          "hive.server2.tez.initialize.default.sessions": "true",
+          "hive.server2.transport.mode": "http"
+        }
+      },
+      {
+        "mapred-site": {
+          "mapreduce.job.reduce.slowstart.completedmaps": "0.7",
+          "mapreduce.map.output.compress": "true",
+          "mapreduce.output.fileoutputformat.compress": "true"
+        }
+      },
+      {
+        "tez-site": {
+          "tez.runtime.shuffle.parallel.copies": "4",
+          "tez.runtime.enable.final-merge.in.output": "false",
+          "tez.am.am-rm.heartbeat.interval-ms.max": "3100"
+        }
+      },
+      {
+        "yarn-site": {
+          "yarn.timeline-service.entity-group-fs-store.active-dir": "file:///hadoopfs/fs1/apps/ats/active",
+          "yarn.timeline-service.entity-group-fs-store.done-dir": "file:///hadoopfs/fs1/apps/ats/done",
+          "yarn.log-aggregation.file-formats": "TFile",
```

Comment:
See above, `"yarn.log-aggregation.fs-support-append": "false"` is probably also needed like for the attached EDW.